airflow/www/static/js/dag/Main.tsx

```diff
@@ -65,24 +70,83 @@ const Main = () => {
     onToggle();
   };
 
+  useEffect(() => {
+    if (contentRef.current) {
+      const topOffset = contentRef.current.offsetTop;
+      const footerHeight = parseInt(getComputedStyle(document.getElementsByTagName('body')[0]).paddingBottom.replace('px', '').replace('em', ''), 10) || 0;
+      contentRef.current.style.height = `${window.innerHeight - topOffset - footerHeight}px`;
+    }
+  }, []);
+
+  const resize = useCallback((e: MouseEvent) => {
+    const gridEl = gridRef.current;
+    if (gridEl && e.x > 350 && e.x < window.innerWidth - 400) {
```

Comment:
this 350 and 400, can they be extracted into variable, at least they have an explicit name. (And can be later reused)